app/Models/Todo.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Todo extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['title', 'excerpt', 'body', 'category_id', 'user_id'];
+
+    public function category()
+    {
+        return $this->belongsTo(Category::class);
+
+    }
+
+    public function user()
+    {
+        return $this->belongsTo(User::class);
+
+    }
```

Comment:
This part isn't relevant to the TODO app, it was added when following the Laracast tutorial